Package.swift

```diff
@@ -122,6 +122,7 @@ let package = Package(
             name: "PackageDescription",
             swiftSettings: [
                 .define("PACKAGE_DESCRIPTION_4_2"),
+                .unsafeFlags(["-package-description-version", "5.5"])
```

Comment:
I'm a bit concerned that this will become a problem.  We will anyway need to add `-library-evolution` here in order to properly build this with SwiftPM, but it does mean that clients can't depend on this library as a dependency (they will have to invoke `swift build` directly on the SwiftPM package, so that the unsafe flags are allowed).